src/interfaces/permissions/request/method.ts

```diff
@@ -0,0 +1,40 @@
+import type { MethodHandler } from '../../types';
+import type { JsonPermissionsRequest } from './types';
+
+import { MessageReply } from '../../../response';
+import { PermissionsRequest } from './message';
+
+export const processPermissionsRequest: MethodHandler = async (
+  ctx,
```

Comment:
Consider:
```suggestion
  context,
```

Reply:
such a strong habit because: https://koajs.com/#context

Reply:
Just making sure: so you prefer to stay with `ctx` 'coz it's such a special common abbreviation that deserves special treatment? = ]
Wanna be 100% sure, 'coz I don't see it being that special.

Reply:
🤣 moreso saying that this won't be the last time you'll be reminding me because `ctx` is etched into my brain. `ctx` is also allll over the codebase 😭

Reply:
1. This is "just" a consistency nit. and
1. sounds like you can't be bothered, and
1. it's "just" a variable name (not a class/type name)

So not a big deal really. Will resolve.